src/goto-symex/symex_decl.cpp

```diff
@@ -27,8 +27,8 @@ void goto_symext::symex_decl(statet &state)
 
   // two-operand decl not supported here
   // we handle the decl with only one operand
-  PRECONDITION(code.operands().size()==1);
-  PRECONDITION(code.op0().id()!=ID_symbol);
+  PRECONDITION(code.operands().size() == 1);
+  PRECONDITION(code.op0().id() == ID_symbol);
```

Comment:
Why has this PRECONDITION changed from != to == ? I don't see why this patch hunk is here as apart from that operator changing, it's just whitespace formatting changing, which is not related to this pr.

Reply:
The original code for this was:

```
if(code.op0().id()!=ID_symbol)	
    throw "decl expects symbol as first operand";
```

When we turn these if conditions into `PRECONDITION`s, we are changing from checking if something shouldn't happen, to checking that the opposite should always happen. I hadn't made the reversal of the condition properly in one of the previous commits, and this compensates for that.